libmscore/score.cpp

```diff
@@ -2887,24 +2887,37 @@ void Score::padToggle(Pad n, const EditData& ed)
       std::vector<ChordRest*> crs;
 
       if (selection().isSingle()) {
-            // do not allow to add a dot on a full measure rest
-            Element* e = selection().element();
-            if (e && e->isRest()) {
-                  Rest* r = toRest(e);
-                  if (r->isFullMeasureRest())
-                        _is.setDots(0);
-                  }
 
-            // on measure rest, select the first actual rest
-            ChordRest* cr = selection().cr();
-            if (cr && cr->isRest() && cr->measure()->isMMRest()) {
-                  Measure* m = cr->measure()->mmRestFirst();
-                  if (m)
-                        cr = m->findChordRest(m->tick(), 0);
-                  }
+            Element* selectedElement = selection().element();
+
+            if (!selectedElement->isNote()) {
+
+                  ed.view->startNoteEntryMode();
+                  deselect(selectedElement);
+
+                  } else {
+
+                  // do not allow to add a dot on a full measure rest
+                  if (selectedElement && selectedElement->isRest()) {
+                        Rest* r = toRest(selectedElement);
+                        if (r->isFullMeasureRest())
+                              _is.setDots(0);
+                        }
 
-            if (cr)
-                  crs.push_back(cr);
+                  // on measure rest, select the first actual rest
+                  ChordRest* cr = selection().cr();
+
+                  if (cr && cr->isRest() && cr->measure()->isMMRest()) {
+                        Measure* m = cr->measure()->mmRestFirst();
+                        if (m) {
+                              cr = m->findChordRest(m->tick(), 0);
+                              }
```

Comment:
Brackets are not needed here. Same for [lines 2917 to 2919](https://github.com/musescore/MuseScore/pull/5469/files#diff-da9ebe454eaee33caed3ff939a69dbaeR2917).

Reply:
thx, fixed